package.json

```diff
@@ -17,6 +18,8 @@
   },
   "devDependencies": {
     "eslint": "^6.8.0",
-    "eslint-config-google": "^0.14.0"
+    "eslint-config-google": "^0.14.0",
```

Comment:
Can we remove eslint-config-google as we're using the prettier config now?

Reply:
You're right. I'll remove it